From 0cbf78b5b02c57e6fd0e57e811cfe56509c4fd24 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Sun, 14 Jul 2013 21:10:37 +0800
Subject: [PATCH] ENGR00269945: reset: build in CONFIG_RESET_GPIO by default

GPIO is widely used as the reset control for various devices.  Let's
build the support in by default.

[shawn.guo: cherry-pick commit 795fcb3bc5bb from imx_3.10.y]
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 drivers/reset/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/reset/Kconfig b/drivers/reset/Kconfig
index 1a862df..99e43ce 100644
--- a/drivers/reset/Kconfig
+++ b/drivers/reset/Kconfig
@@ -16,6 +16,7 @@ if RESET_CONTROLLER
 
 config RESET_GPIO
 	tristate "GPIO reset controller support"
+	default y
 	depends on GPIOLIB && OF
 	help
 	  This driver provides support for reset lines that are controlled
-- 
1.8.0

